src/components/inputs/money-input/money-input.styles.js

```diff
@@ -29,4 +29,20 @@ const getAmountInputStyles = props => [
   `,
 ];
 
-export { getCurrencyLabelStyles, getAmountInputStyles };
+const getHighPrecisionWrapperStyles = props => css`
```

Comment:
Nit just pass `isDisabled`? to be more specific? Not sure what the pattern is on UIKit.

Reply:
👍